system/lib/compiler-rt/lib/sanitizer_common/sanitizer_emscripten.cpp

```diff
@@ -127,6 +127,8 @@ u64 MonotonicNanoTime() {
   return (u64)ts.tv_sec * (1000ULL * 1000 * 1000) + ts.tv_nsec;
 }
 
+void GetMemoryProfile(fill_profile_f cb, uptr *stats) {}
```

Comment:
This is needed for wasm64 but not wasm32 because its referenced by `compiler-rt/lib/sanitizer_common/sanitizer_allocator_primary64.h` but not by `compiler-rt/lib/sanitizer_common/sanitizer_allocator_primary32.h`